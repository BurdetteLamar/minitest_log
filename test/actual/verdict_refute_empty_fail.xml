<log>
  <summary_ verdicts='3' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_refute_empty?' type='fail'>
    <section_ name='Args=[[], &quot;Message 0&quot;]'>
      <verdict_ method='verdict_refute_empty?' outcome='failed' id='0' message='Message 0'>
        <actual_ class='Array' value='[]'/>
        <exception_ class='Minitest::Assertion' message='Expected [] to not be empty.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[&quot;&quot;, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_refute_empty?' outcome='failed' id='1' message='Message 1'>
        <actual_ class='String' value='&quot;&quot;'/>
        <exception_ class='Minitest::Assertion' message='Expected # encoding: UTF-8'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
