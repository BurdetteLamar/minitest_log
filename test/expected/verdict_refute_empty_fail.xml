<log>
  <summary_ verdicts='2' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_refute_empty?' type='fail'>
    <section_ name='Args=[[]]'>
      <verdict_ method='verdict_refute_empty?' outcome='failed' id='0' message='Message 0'>
        <actual_ class='Array' value='[]'/>
        <exception_ class='Minitest::Assertion' message='Expected [] to not be empty.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[&quot;&quot;]'>
      <verdict_ method='verdict_refute_empty?' outcome='failed' id='1' message='Message 1'>
        <actual_ class='String' value='&quot;&quot;'/>
        <exception_ class='Minitest::Assertion' message='Expected # encoding: UTF-8'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
  </section_>
</log>
